### PR TITLE
[0.6 Release][Object Detection][Tutorial] Remove mmdet output format

### PR DESCRIPTION
**Pull request by yongxinw**
*Issue #, if available:*

*Description of changes:* Update tutorials to reflect the pd.DataFrame output from predictor for object detection

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md [hunk 1]
```diff
@@ -63,54 +69,28 @@ To run inference, perform:
 
 ```{.python}
 pred = predictor.predict(test_path)
+print(pred)
```
Comment (FANGAreNotGnu) [inline docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md:72]: Output is not pasted here

Comment (yongxinw) [reply docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md:72]: oh got it. Let me paste the outputs

**Review by bryanyzhu — COMMENTED** [review]
Thank you for the PR, I Left a few comments

File: docs/tutorials/multimodal/object_detection/inference/detection_inference_quick_start.md [hunk 2]
```diff
@@ -1,42 +1,53 @@
...
+# AutoMM Detection - Inference Quick Start with Pretrained VFNet
 :label:`sec_automm_detection_infer_tiny_motorbike`
 
-In this section, we show an quick-start example to run inference on a small dataset (Tiny Motorbick) that is in COCO Format. 
+In this section, we show an quick-start example to run inference on a single image and visualize the detections.  
 The model we use is the VFNet pretrained on COCO dataset.
 
 ## Prepare data
-```{.python}
+```{.python .input}
```
[the remark this answers is not in the document]
Comment (yongxinw) [reply docs/tutorials/multimodal/object_detection/inference/detection_inference_quick_start.md:8]: ok

File: docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md [hunk 3]
```diff
@@ -5,9 +5,15 @@ In this section, we show an example to run inference COCO dataset in COCO Format
 Different from running evaluation, the purpose is to get detection results for potential down-stream tasks.
 The model we use is the VFNet pretrained on COCO dataset.
 
+AutoMM detection requires `mmcv-full` and `mmdet` packages. Please make sure `mmcv-full` and `mmdet` are installed:
+```{.python}
```
Comment (sxjscience) [inline docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md:9]: Change to {.python .input}

Comment (sxjscience) [reply docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md:9]: Ignore the previous comment. should change to \`\`\`python

**Review by bryanyzhu — APPROVED** [review]
LGTM

**Review by FANGAreNotGnu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**